### PR TITLE
Update URL to be compatible with brew bump-formula-pr

### PR DESCRIPTION
**Pull request by eculver**
the `brew bump-formula-pr` helper that I'm going to be calling in a Github Action to automatically bump this, depends on the version matching a certain taxonomy for future versions to be updated. This should make it line up.

### CODE REVIEW

**Comment by eculver** [comment]
I also ran `brew style --fix` on the formulas.

File: Formula/supergrain-cli.rb [hunk 1]
```diff
@@ -158,7 +157,9 @@ class SupergrainCli < Formula
   end
 
   def install
-    unless build.without?("snowflake-connector")
+    if build.without?("snowflake-connector")
```
Comment (eculver) [inline Formula/supergrain-cli.rb:160]: these are from `brew style --fix`

Comment (lucasmarshall) [reply Formula/supergrain-cli.rb:160]: 👍

**Review by lucasmarshall — APPROVED** [review]
LGTM

**Pull request merged**